.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:17.03-git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.03.0-ce
+          docker_layer_caching: true
+  python_unit_test:
+    docker:
+      - image: circleci/python:2.7
+        environment:
+          DJANGO_SETTINGS_MODULE: cla_frontend.settings.testing
+    steps:
+      - checkout
+      - run:
+          name: Setup Python environment
+          command: |
+            pip install virtualenv
+            virtualenv env-ci
+      - restore_cache:
+          keys:
+            - pip-v2-{{ checksum "requirements/testing.txt" }}
```

Comment:
_nitpick_ We could use `-v1` as this is the first instance of this script.

Reply:
Using `checksum "requirements/testing.txt"` will not detect changes in `base.txt`:

```
cla_frontend$ cat requirements/testing.txt
-r base.txt

mock==1.0.1
git+git://github.com/tomchristie/django-pdb.git@e28e9ee73e52e7df4a125ec7326882e5d8e83740#egg=django-pdb==0.3.2-dev
```

Could we use `-{{ checksum "requirements/base.txt" }}` or `-{{ checksum "requirements/base.txt" }}-{{ checksum "requirements/testing.txt" }}` instead, please?

Reply:
Will do.

We should start using `pipenv` soon. See https://circleci.com/docs/2.0/caching/#pip-python

Reply:
Pipenv would be ideal 👍